res/values/strings.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <!-- common strings without special context -->
-    <string name="app_name">Delta Chat</string>
+    <string name="dc_app_name" translatable="false">Delta Chat</string>
```

Comment:
why was this string  added?

Reply:
I deleted untranslated.xml. There was only this. So I moved those over here.

Reply:
Ok, I've found it in the untranslated.xml. Instead of adding this entry here, iOS needs to use the right string "app_name"

Reply:
It does not hurt the iOS string. I did it to get out of the crowd. Would you like undone?

Reply:
Yes, you can delete the entry for dc_app_name but keep app_name. dc_app_name is not used in iOS (anymore)

Reply:
Does Android use it? I did it for Android.

Reply:
yes it is used there, but in a separate untranslated.xml file. In addition to that, I'll remove that and use only app_name instead on Android as well.